### PR TITLE
Feat/change params

### PR DESCRIPTION
**Pull request by GustavoTCh**
Done:
- Migration packages( latlng , marker)
- Adding new tags banned in Park and ride 
- Other package migrations that run errors in stadnaviing error in stadnavi

### CODE REVIEW

**Comment by GustavoTCh** [comment]
@SamuelRioTz The updates of the packages in trufi-core are creating some errors, I do not know if it is necessary to create another PR to solve these new problems

File: pubspec.yaml [hunk 1]
```diff
@@ -17,13 +17,11 @@ dependencies:
   flutter:
     sdk: flutter
 
-  graphql: ^4.0.1
+  # graphql: ^5.0.0
```
Comment (SamuelRioTz) [inline pubspec.yaml:20]: please remove commented code

Comment (GustavoTCh) [reply pubspec.yaml:20]: okey

**Comment by GustavoTCh** [comment]
@SamuelRioTz  can you update the ref for new release?

**Review by SamuelRioTz — APPROVED** [review]

**Pull request merged**